src/Writer/ExtensionPluginManager.php

```diff
@@ -22,29 +22,30 @@ class ExtensionPluginManager extends AbstractPluginManager implements ExtensionM
     /**
      * Aliases for default set of extension classes
      *
-     * @var array
+     * @var array<array-key, string>
```

Comment:
`array<string, ...` rather than `array<array-key, `

Reply:
Parent is defined `array<array-key, string>`.